content/languages.yml

```diff
@@ -0,0 +1,124 @@
+- name: Arabic
+  translated_name: العربية
+  code: ar
+  is_completed: false
+- name: Azerbaijani
+  translated_name: Azərbaycanca
+  code: az
+  is_completed: false
+- name: Bulgarian
+  translated_name: Български
+  code: bg
+  is_completed: false
+- name: Bengali
+  translated_name: বাংলা
+  code: bn
+  is_completed: false
+- name: German
+  translated_name: Deutsch
+  code: de
+  is_completed: false
+- name: Spanish
+  translated_name: Español
```

Comment:
English goes here!

Reply:
We could ask the translations to do it (add english and remove their language from the list) but that’s a lot of redundant effor